### ISSUE TITLE
flutter 3 There is warning

### ISSUE THREAD

**Issue by purgeyao** [mingsnx/animated_digit#5]
```
SchedullerBinding.instance!.addPostFrameCallback(...)
                          ^
                          |---- Since flutter 3.0.0 instance does not nullable.
```

**Comment by mingsnx** [comment]
OK

**Comment by mingsnx** [comment]
( •̀ ω •́ )[published v3.2.0](https://pub.flutter-io.cn/packages/animated_digit)
